build-system/tasks/extension-helpers.js

```diff
@@ -366,16 +367,24 @@ function buildExtension(
       buildExtension(name, version, latestVersion, hasCss, optionsCopy);
     });
   }
-  let promise = Promise.resolve();
+  const promises = [];
   if (hasCss) {
     mkdirSync('build');
     mkdirSync('build/css');
-    promise = buildExtensionCss(path, name, version, optionsCopy);
+    const buildCssPromise = buildExtensionCss(path, name, version, optionsCopy);
     if (options.compileOnlyCss) {
-      return promise;
+      return buildCssPromise;
     }
+
+    promises.push(buildCssPromise);
   }
-  return promise.then(() => {
+
+  // minify and copy vendor configs for amp-analytics component
+  if (name === 'amp-analytics' && argv.compile_vendor_configs) {
```

Comment:
I was thinking about a gulp task called `compile_vendor_configs` that runs individually of `gulp build`. The reason for that is we could later combine the task along with the task to convert config file to json easily. What do you think. I'm also cool with this, as long as it's easy to run in the release process.

Reply:
This is already contained in its own gulp task called `gulp vendor-configs` so combining this with the JSON conversion gulp task will be easy.

I think after we release this feature we will want to run this task here when compiling `amp-analytics` so we should keep this here.